### ISSUE TITLE
Alfresco 7.2 Bitnami VM

### ISSUE THREAD

**Issue by pablodazevedo** [bitnami/vms#345]
### Describe your issue as much as you can

Alfresco Search Services doesn't works in a new Appliance.
I have tested the installation in various ways but I can't find the problem.

Site searchs and Searchs bar does't works.
The applets "Image views", "Recent Documents" and that look for something in the repository don't find anything.

"Image View" message - Site Sample Web Site Design Project
08200004 Wrapped Exception (with status template): 08200023 Failed to execute script 'classpath*:alfresco/templates/webscripts/org/alfresco/slingshot/documentlibrary/images.get.js': 08200022 Failed to execute search: +PATH:"/app:company_home/st:sites/cm:swsdp/cm:documentLibrary//*" +TYPE:"cm:content" +@cm\:content.mimetype:image/*

Can someone help me?

**Comment by mdhont** [comment]
Can you describe the exact steps to reproduce the issue?

**Comment by pablodazevedo** [comment]
I didn't do any special steps. I simply imported Alfresco Bitnami appliance in Virtual Box, checked the network and firewall, and ran the service Bitnami or the ctlscript.sh script.
When accessing Alfresco Share, the Alfresco searchs didn't work, the Image View dashlet showed the above message and other content related dashlets had strange behavior.
After so much studying, I realized that the Alfresco Bitnami application didn't created the Solr Cores "alfresco" and "archive".
I decided to reinstall Alfresco Search Services and finally it fixed these problems.
I managed to make it functional, but the Solr service I have to deal with separately from the other services that are automated via Bitnami scripts.
